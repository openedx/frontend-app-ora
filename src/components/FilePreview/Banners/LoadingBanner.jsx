import React from 'react';

import { Alert, Spinner } from '@edx/paragon';

export const LoadingBanner = () => (
  <Alert variant="info">
    <Spinner animation="border" className="d-flex m-auto" />
  </Alert>
);

LoadingBanner.defaultProps = {};
LoadingBanner.propTypes = {};

export default LoadingBanner;
