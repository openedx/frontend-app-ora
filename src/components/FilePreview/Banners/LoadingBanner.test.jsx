import React from 'react';
import { shallow } from '@edx/react-unit-test-utils';

import LoadingBanner from './LoadingBanner';

describe('Loading Banner component', () => {
  test('snapshot', () => {
    const el = shallow(<LoadingBanner />);
    expect(el.snapshot).toMatchSnapshot();
  });
});
