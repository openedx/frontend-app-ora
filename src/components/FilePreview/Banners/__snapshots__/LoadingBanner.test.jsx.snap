// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Banner component snapshot 1`] = `
<Alert
  variant="info"
>
  <Spinner
    animation="border"
    className="d-flex m-auto"
  />
</Alert>
`;
